### PR TITLE
Update llama.cpp integration

### PR DESCRIPTION
**Pull request by ElliotKetchup**

### CODE REVIEW

File: docs/docs/integrations/llms/llamacpp.ipynb
Comment (baskaryan) [inline docs/docs/integrations/llms/llamacpp.ipynb:1]: believe it supports more than just Llama models https://github.com/ggerganov/llama.cpp

File: docs/docs/integrations/llms/llamacpp.ipynb [hunk 1]
```diff
@@ -6,9 +6,9 @@
    "source": [
     "# Llama.cpp\n",
     "\n",
-    "[llama-cpp-python](https://github.com/abetlen/llama-cpp-python) is a Python binding for [llama.cpp](https://github.com/ggerganov/llama.cpp). \n",
+    "[llama-cpp-python](https://github.com/abetlen/llama-cpp-python) is a Python binding for [llama.cpp](https://github.com/ggerganov/llama.cpp).\n",
     "\n",
-    "It supports inference for [many LLMs](https://github.com/ggerganov/llama.cpp), which can be accessed on [HuggingFace](https://huggingface.co/TheBloke).\n",
+    "It supports inference for Meta's [LLaMA](https://github.com/facebookresearch/llama/) models, which can be accessed on [HuggingFace](https://huggingface.co/TheBloke).\n",
```
Comment (leo-gan) [inline docs/docs/integrations/llms/llamacpp.ipynb:11]: llama.cpp supports many LLMs (I've copied a text from the site). 
"Supported models:

 LLaMA 🦙
 LLaMA 2 🦙🦙
 Falcon
 [Alpaca](https://github.com/ggerganov/llama.cpp#instruction-mode-with-alpaca)
 [GPT4All](https://github.com/ggerganov/llama.cpp#using-gpt4all)
 [Chinese LLaMA / Alpaca](https://github.com/ymcui/Chinese-LLaMA-Alpaca) and [Chinese LLaMA-2 / Alpaca-2](https://github.com/ymcui/Chinese-LLaMA-Alpaca-2)
 [Vigogne (French)](https://github.com/bofenghuang/vigogne)
 https://github.com/ggerganov/llama.cpp/discussions/643#discussioncomment-5533894
 [Koala](https://bair.berkeley.edu/blog/2023/04/03/koala/)
 [OpenBuddy 🐶 (Multilingual)](https://github.com/OpenBuddy/OpenBuddy)
 [Pygmalion/Metharme](https://github.com/ggerganov/llama.cpp#using-pygmalion-7b--metharme-7b)
 [WizardLM](https://github.com/nlpxucan/WizardLM)
 [Baichuan 1 & 2](https://huggingface.co/models?search=baichuan-inc/Baichuan) + [derivations](https://huggingface.co/hiyouga/baichuan-7b-sft)
 [Aquila 1 & 2](https://huggingface.co/models?search=BAAI/Aquila)
 https://github.com/ggerganov/llama.cpp/pull/3187
 [Mistral AI v0.1](https://huggingface.co/mistralai/Mistral-7B-v0.1)
 [Refact](https://huggingface.co/smallcloudai/Refact-1_6B-fim)
 https://github.com/ggerganov/llama.cpp/pull/3410
 https://github.com/ggerganov/llama.cpp/pull/3417
 https://github.com/ggerganov/llama.cpp/pull/3553"

**Review by leo-gan — CHANGES_REQUESTED** [review]
Could be a misunderstanding.

File: docs/docs/integrations/llms/llamacpp.ipynb (hunk 1 above)
Comment (ElliotKetchup) [reply docs/docs/integrations/llms/llamacpp.ipynb]: I agree and reverted the line. The only change is that the link now redirects to the Description of llama.cpp's README

**Review by efriis — APPROVED** [review]

**Comment by efriis** [comment]
Thanks @ElliotKetchup !

**Pull request merged**